src/views/bdsp/main-view.hpp

```diff
@@ -25,7 +26,7 @@ class MainBdSpView : public tsl::Gui {
     auto list = new tsl::elm::List();
 
     list->addItem(new tsl::elm::CategoryHeader("Pokemon"));
-    list->addItem(new PokemonViewButton("Wild", csight::bdsp::read_wild_pokemon));
+    list->addItem(new EncounterListViewButton(csight::bdsp::read_encounter_pokemon));
     list->addItem(new PartyListViewButton(csight::bdsp::read_party_pokemon));
```

Comment:
With the above changes, these lines to should be updated to:
```cpp
u32 encounter_count = csight::bdsp::read_encounter_pokemon_count();
list->addItem(new PokemonListViewButton("Encounters", csight::bdsp::read_encounter_pokemon, encounter_count));
list->addItem(new PokemonListViewButton("Party", csight::bdsp::read_party_pokemon, 6));
```

While not a requirement, if you're interested you should be able to create `read_party_pokemon_count` using code similar to the first comment.  If you're not interested, no worries.